dotnet/src/dotnetcore/Providers/Messaging/GXAmazonSQS/GXAmazonSQS.csproj

```diff
@@ -1,7 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
+    <PackageId>GeneXus.AWS.Queue</PackageId>
+    <PackageTags>Message Amazon</PackageTags>
```

Comment:
Maybe "AWS SQS Queue Messaging" for better discoverability?

Reply:
Done.